app/controllers/staff/proposals_controller.rb

```diff
@@ -1,6 +1,8 @@
 class Staff::ProposalsController < Staff::ApplicationController
   before_action :enable_staff_program_subnav
   before_action :set_proposal_counts, only: [:index, :show, :selection]
+  before_action :skip_policy_scope
```

Comment:
The only place policy_scope is used was a test I did on the reviewer proposal index. It doesn't just happen automatically.